src/features/search/searchAtoms.ts

```
@@ -0,0 +1,141 @@
+export const highlightedIndexAtom = atom(-1, 'highlightedIndexAtom');
+
+export type AggregatedSearchItem =
+  | (MCDAConfig & { source: 'mcda' })
+  | (GeoJSON.Feature<Geometry, LocationProperties> & { source: 'locations' });
+
+export const aggregatedSearchAtom = atom((ctx) => {
+  const locationsData = ctx.spy(fetchLocationsAsyncResource.dataAtom);
+  const mcdaData = ctx.spy(fetchMCDAAsyncResource.dataAtom);
+
+  const results: AggregatedSearchItem[] = [];
+  if (mcdaData && mcdaData.config) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)